doc/en/developer/source/policies/patches.rst

```diff
@@ -9,6 +9,25 @@ is a great way to give back to the project.
 
 This document is a guide designed to help users through the process of successfully submitting a patch.
 
+.. note:: 
+
+   We would like to see all fixes applied to master first, and then back ported to the stable and maintenance branches. 
+   
+   We require the following for fixes submitted via pull requests or patch:
```

Comment:
This has been merged, want to take this to email and I can clean up this section to only talk about pull requests.